Lib/test/test_perf_profiler.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+import subprocess
+import re
+import sys
+import sysconfig
+import os
+from test import support
+from test.support.script_helper import make_script
+from test.support.os_helper import temp_dir
+from test.support import check_sanitizer
+
+
+def get_perf_version():
+    try:
+        cmd = ["perf", "version"]
+        proc = subprocess.run(
+            cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True
+        )
+    except (subprocess.SubprocessError, OSError):
+        raise unittest.SkipTest("Couldn't find perf on the path")
+    
+    version = proc.stdout
+
+    match = re.search(r"^perf version\s+(.*)", version)
+    if match is None:
+        raise Exception("unable to parse perf version: %r" % version)
+    return (version, match.group(1))
+
+if not support.has_subprocess_support:
+    raise unittest.SkipTest("test module requires subprocess")
+
+
+_, version = get_perf_version()
+
+if not version:
+    raise unittest.SkipTest("Could not find valid perf tool")
+
+if "no-omit-frame-pointe" not in sysconfig.get_config_var("CFLAGS"):
```

Comment:
```suggestion
if "no-omit-frame-pointer" not in sysconfig.get_config_var("CFLAGS"):
```

Reply:
Already addressed in https://github.com/pablogsal/cpython/pull/37